TicketBookingSystemTest/BusinessLayerTest/CountryTest.cs

```diff
@@ -0,0 +1,45 @@
+﻿using AutoFixture;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System.ComponentModel.DataAnnotations;
+
+namespace TicketBookingSystemTest.BusinessLayerTest
+{
+    [TestClass]
+    public class CountryTests
+    {
+        private Fixture fixture = new Fixture();
+
+        [TestInitialize]
+        public void TestInitialize()
+        {
+            fixture = new Fixture();
+            fixture.Customize(new RandomRangedNumberCustomization());
+        }
+        [TestMethod]
+        public void FillFromStrings_ShouldCreateCountryFromValues()
+        {
+            // Arrange
+            fixture.RepeatCount = 2;
+            var values = fixture.Create<string[]>(); 
```

Comment:
Do fixtures generate a valid CountryCode and CountryName? assume not, how this test pass?